### ISSUE TITLE
MKL_DNN License File missing 

### ISSUE THREAD

**Issue by sadatnfs** [tensorflow/tensorflow#20811]
Hello, 

It seems like since commit #20576 (specific PR : [here](https://github.com/tensorflow/tensorflow/commit/03ab64c3a68f3b990bf690ede06e3066ad4e35a0#diff-2f07a0589130c83666437024dd03d252)), building TF with MKL option forces the configure step to look for a LICENSE file inside the `third_party/mkl_dnn` folder, which actually doesn't exist. 

Currently, I'm using this hack in my Docker for the build to proceed without needing the explicit DNN LICENSE file (by copying the MKL license file into the MKL_DNN folder since they are identical licenses), but it'd be great if we could have this fixed sometime? I opened a small PR with this small fix (copied the LICENSE file from the original MKL-DNN repo in #20810 (identical to MKL license, but just for pedantic reasons)) so maybe this will help other users out as well.

## Environment Info:
Have I written custom code: No
OS Platform and Distribution: Tested on Debian:Stretch
TensorFlow installed from: Source
TensorFlow version:  1.9.0 (Master branch)
Bazel version: 0.15.0
CUDA/cuDNN version: N/A
GPU model and memory: N/A
Exact command to reproduce: 
```
d /opt && \
    git clone https://github.com/tensorflow/tensorflow.git && \
    cd /opt/tensorflow && \
    cp /opt/tensorflow/third_party/mkl/LICENSE /opt/tensorflow/third_party/mkl_dnn/LICENSE && \
    /bin/bash ./configure \
    && \
    bazel build --config=opt --config=mkl \
    tensorflow/tools/pip_package:build_pip_package
```

Thanks!

Nafis

**Comment by tensorflowbutler** [comment]
Thank you for your post. We noticed you have not filled out the following field in the issue template. Could you update them if they are relevant in your case, or leave them as N/A? Thanks.
Have I written custom code
OS Platform and Distribution
TensorFlow installed from
TensorFlow version
Bazel version
CUDA/cuDNN version
GPU model and memory
Exact command to reproduce

**Comment by sadatnfs** [comment]
Added the info to my main post, thanks!

**Comment by reedwm** [comment]
Has this issue been fixed by #20810? Should I close it?

**Comment by sadatnfs** [comment]
Yep it's fixed!
